documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -895,13 +895,16 @@ engines running on the JUnit Platform via one of the following mechanisms.
     `systemProperty` or `systemProperties` DSL.
   * <<running-tests-build-maven-config-params,Maven Surefire provider>>: use the
```

Comment:
The new `--config-resource` option should also be included in this bullet point listing, below the `--config` discussion or incorporated into that discussion about the Console Launcher.

Reply:
Added 👍